mne/evoked.py

```diff
@@ -559,8 +560,8 @@ def __neg__(self):
         out.comment = '-' + (out.comment or 'unknown')
         return out
 
-    def get_peak(self, ch_type=None, tmin=None, tmax=None, mode='abs',
-                 time_as_index=False):
+    def get_peak(self, ch_type=None, merge_grads=False, tmin=None, tmax=None,
```

Comment:
Ahh, I though it wouldn't matter since they are kwargs, but of course it does. Silly me.